prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -259,58 +265,57 @@ struct fi_ibv_rdm_cntr {
 
 struct fi_ibv_rdm_ep {
 	struct fid_ep ep_fid;
-	struct fi_ibv_domain *domain;
-	struct fi_ibv_rdm_cq *fi_scq;
-	struct fi_ibv_rdm_cq *fi_rcq;
-
-	struct fi_ibv_rdm_cntr *send_cntr;
-	struct fi_ibv_rdm_cntr *recv_cntr;
-	struct fi_ibv_rdm_cntr *read_cntr;
-	struct fi_ibv_rdm_cntr *write_cntr;
-
-	size_t addrlen;
-	struct rdma_addrinfo *rai;
-	struct sockaddr_in my_addr;
-
-	struct fi_ibv_av *av;
-	int tx_selective_completion;
-	int rx_selective_completion;
-	size_t min_multi_recv_size;
-	uint64_t tx_op_flags;
-	uint64_t rx_op_flags;
+	struct fi_ibv_domain*	domain;
+	struct slist_entry	list_entry;
+	struct fi_ibv_rdm_cq*	fi_scq;
+	struct fi_ibv_rdm_cq*	fi_rcq;
```

Comment:
This is a total nit, but the '*' is usually placed next to the variable/field name, not the structure name.  There are several areas throughout this series where this is done, but I'm not sure it's worth re-spinning the series just for this change.